data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,458 @@
+}
+
+tribute_not_started = {
+   {
+      title = _"Urgent Tribute Demand",
+      body = kalitath(
+         -- TRANSLATORS: Kalitath
+         _([[Strangers, we haven't received any help yet. So if you are really on our side you should start sending some wares urgently.]])..
+         _([[As we are facing great losses due to your lazyness, you should now send us double the amount of each ware as before.]])
+         .. paragraphdivider() ..
+         _([[I am sure you won't loose our alliance, wouldn't you?]]))
+   },
+   {
+      title = _"Blackmail!",
+      body = sidolus(
+         -- TRANSLATORS: Sidolus
+         _([[May Satul warm us all. Our so-called ally is blackmailing us and we can't punish his insult as he has our princess at his hands.]])
+         .. paragraphdivider() ..
+         _([[Our only chance is to double our efforts and start delivering something soon.]]))
+         .. new_objectives(obj_tribute2)
+   },
+}
+
+tribute_started = {
+   {
+      title = _"Tribute Started",
+      body = kalitath(
+         -- TRANSLATORS: Kalitath
+         _([[Strangers, we haven't received the first batch of wares. So we may finally trust you.]])..
+         _([[However our enemy in the north is very strong. I am afraid without significant help from your side we might loose the war despite any ware help you might provide.]])
+         .. paragraphdivider() ..
+         _([[They name themselves an empire and their leader is named Maletus. I want you to train your soldiers and help us defeat his lousy empire.]]))
+         .. new_objectives(obj_defeat_maletus)
+   },
+   {
+      title = _"Be Vigilant!",
+      body = jundlina(_"Jundlina",
+         -- TRANSLATORS: Jundlina
+         _([[May Satul warm you, Sidolus. Our ally now has gained some trust, but we shouldn't trust to much in them.]])
+         .. paragraphdivider() ..
+         _([[So I want you to defeat this empire but we should be strong enough to get the respect we earn.]]))
+   },
+}
+
+alliance_broken = {
+   {
+      title = _"Traitors",
+      body = kalitath(
+         -- TRANSLATORS: Kalitath
+         _([[Strangers, we still haven't received any help. We are deeply insulted by your treachery.]])..
+         _([[At least your princess will make a reasonable slave, if we will survive the battle with our enemy.]]))
+   },
+   {
+      title = _"Jundlinas Suicide",
+      body = jundlina(_"Jundlina",
+         -- TRANSLATORS: Jundlina
+         _([[Barbarian, I am an atlantean princess and I will never go into slavery. I have prepared myself for this eventuality and will rather die from the poison I carry with me then being your slave. May Satul revenge my sacrifice.]]))
+   },
+   {
+      title = _"Defeated!",
+      body = sidolus(
+         -- TRANSLATORS: Sidolus
+         _([[Aaaargh. I don’t know how this could have happened, but Satul couldn’t protect us from the enemy. We have lost our noble princess. So, we have lost this battle and our people is doomed!]]))
```

Comment:
```suggestion
         _([[Aaaargh. I don’t know how this could have happened, but Satul couldn’t protect us from the enemy. We have lost our noble princess. Thus, we have lost this battle and our people is doomed!]]))
```